frontend/controller/serviceworkers/sw-primary.js

```diff
@@ -16,12 +17,11 @@ import notificationGetters from '~/frontend/model/notifications/getters.js'
 import '~/frontend/model/notifications/selectors.js'
 import setupChelonia from '~/frontend/setupChelonia.js'
 import { KV_KEYS } from '~/frontend/utils/constants.js'
-import { CHELONIA_STATE_MODIFIED, LOGIN, LOGIN_ERROR, LOGOUT, LOGGING_OUT } from '~/frontend/utils/events.js'
+import { CHELONIA_STATE_MODIFIED, LOGGING_OUT, LOGIN, LOGIN_ERROR, LOGOUT } from '~/frontend/utils/events.js'
```

Comment:
Why was this change made? 🤔

Reply:
Automated change, but it correctly orders the imports alphabetically.